src/transformStateWithClones.js

```diff
@@ -7,7 +7,39 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const finalRes = [];
```

Comment:
don't shorten variable names if unnecessary, also use something more specific, like `stateClones` for example